examples/advanced/federated-statistics/df_stats.ipynb

```diff
@@ -222,9 +222,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "nvflare_example",
```

Comment:
Should stick with one kernel. Several notebooks are still using nvflare_example.